pyproject.toml

```diff
@@ -22,7 +22,7 @@ env_files = [".env"]
 
 [tool.poetry.dependencies]
 python = ">=3.11,<4.0"
-fastapi = ">=0.51,<1.0"
```

Comment:
Why do we narrow down fastapi dependency so badly?
Let's just fixate starlette then 
```
starlette = ">=0.40.0,<1.0"
```

Reply:
`starlette>=0.40.0` implies `fastapi>=0.115.2` - which is actually **the lastest version** of FastAPI.